account_bank_statement_import_ofx/wizard/account_bank_statement_import.py

```diff
@@ -18,6 +20,22 @@
 class AccountBankStatementImport(models.TransientModel):
     _inherit = 'account.bank.statement.import'
 
+    def _check_journal_bank_account(self, journal, account_number):
+        res = super(
+            AccountBankStatementImport, self
+        )._check_journal_bank_account(journal, account_number)
+        if not res:
+            e_acc_num = journal.bank_account_id.sanitized_acc_number
+            e_acc_num = e_acc_num.replace(" ", "")
+            validate_iban(e_acc_num)
```

Comment:
I think you should encapsulate this in try clause because this may raise a ValidationError, in case you want to just return False

Reply:
As everything is already encapsulated with a general try, I think this can be considered OK.